examples/step-79/doc/intro.dox

```diff
@@ -6,23 +6,23 @@ structure that is bearing some load. Ideally, we would like to minimize the
 maximum stress placed on a structure by selecting a region $E$ where material is
 placed. In other words,
 @f[
-  \text{Minimize}\norm{\boldsymbol{\sigma} (\textbf{u}) }_\infty
+  \text{Minimize}\norm{\boldsymbol{\sigma} (\mathbf{u}) }_\infty
 @f]
 @f[
   \text{subject to } |E|\leq V_{\max},
 @f]
 @f[
-  \text{and } \nabla \cdot \boldsymbol{\sigma} +\textbf{ F} = \mathbf{0}.
+  \text{and } \nabla \cdot \boldsymbol{\sigma} + \mathbf{F} = \mathbf{0}.
 @f]
-Here, $\sigma=C\varepsilon(\mathbf u)$ is the stress within the body that is
+Here, $\boldsymbol{\sigma} = \mathbf{C} : \boldsymbol{\varepsilon}(\mathbf{u})$ is the stress within the body that is
 caused by the external forces $\mathbf F$, where we have for simplicity assumed
-that the material is linear-elastic and so $C$ is the stress-strain tensor and
-$\varepsilon(\mathbf u)=\frac 12 (\nabla \mathbf u + \nabla\mathbf u^T)$ is the
-small-deformation strain as a function of the displacement $\mathbf u$ -- see
+that the material is linear-elastic and so $\mathbf{C}$ is the stress-strain tensor and
+$\boldsymbol{\varepsilon}(\mathbf{u})=\frac{1}{2} (\nabla \mathbf{u} + (\nabla\mathbf{u})^T)$ is the
```

Comment:
I think that I saw you enclosing the `\nabla\mathbf{u}` in brackets before the `^T`, so I added that here as well. Personally, I feel that it disambiguates the notation which is good.